src/azul/plugins/metadata/anvil/__init__.py

```diff
@@ -277,6 +277,19 @@ def facets(self) -> Sequence[str]:
     def manifest_config(self) -> ManifestConfig:
         result = defaultdict(dict)
 
+        # This is used to modify the field names from how they are specified in
+        # the field mapping. Keys are field paths in an ES hit, and values are
+        # the desired manifest column name, or None to omit from the manifest.
+        custom_field_names = {
+            ('bundles', 'uuid'): 'files.bundle_uuid',
+            ('bundles', 'version'): 'files.bundle_version',
+            ('contents', 'activities', 'activity_table'): None,
+            ('contents', 'files', 'uuid'): None,
+            ('contents', 'files', 'version'): None,
+            ('sources', 'id'): 'sources.source_id',
+            ('sources', 'spec'): 'sources.source_spec',
+        }
```

Comment:
The intent of this translation is to make the column names match the response property paths, so this is a brittle coupling to wherever the latter are defined. Think of a way to alleviate or avoid that brittle coupling and propose that in PL.